src/index.js

```diff
@@ -134,7 +135,8 @@ class Griddle extends Component {
 
     this.store = createStore(
       reducers,
-      initialState
+      initialState,
+      applyMiddleware(...plugins.map(p => p.reduxMiddleware), ...reduxMiddleware) 
     );
```

Comment:
That's very true, I'll flatten them as well!

I compacted it as well (removing undefined from the mapping) because `applyMiddleware` will try to call even undefined functions. See the latest file changes.